clarifai/client/user_api.py

```diff
@@ -0,0 +1,48 @@
+from clarifai_grpc.grpc.api import resources_pb2, service_pb2  # noqa: F401
+
+from clarifai.client.app_api import AppApi
+from clarifai.client.base_auth import BaseAuth
+
+
+class UserApi(BaseAuth):
```

Comment:
I find it a bit repetitive/redundant to emphasize the Api/_api in the naming .e.g. as in `app_api` & `user_api`, should we have them as `apps.py/Apps`, `users.py/Users` so that imports are as `from clarifai.client.apps` or `from clarifai.client.users`?
I missed the design sessions, so it could be that this (user_api, app_api) convention was the chosen one.

Reply:
Naming is not finalised. Now that you say, I agree with removing `api`. But we planned on using respective resource protos, so it should be changed as `user_info` or something